pyproject.toml

```diff
@@ -8,7 +8,7 @@ requires-python = ">=3.8"
 dependencies = [
     "anyascii>=0.3.2",
     "blingfire==0.1.8",
-    "boto3>=1.28",
+    "boto3[crt]>=1.28",
```

Comment:
Since we can't use this on most instance types let's drop it for now.

Reply:
we can make it optional